python/mxnet/gluon/data/vision/datasets.py

```diff
@@ -28,9 +28,9 @@
 import warnings
 import numpy as np
 
-from . import dataset
-from ..utils import download, check_sha1
-from ... import nd, image, recordio
+from .. import dataset
+from ...utils import download, check_sha1
+from .... import nd, image, recordio
```

Comment:
This move caused the links to existing Vision datasets to break. See https://mxnet.incubator.apache.org/versions/master/api/python/gluon/data.html?highlight=vision#vision